packages/javascript/jupyterlab-plotly/src/Figure.ts

```diff
@@ -826,9 +826,13 @@ export class FigureView extends DOMWidgetView {
     this.model.on("change:_py2js_update", this.do_update, this);
     this.model.on("change:_py2js_animate", this.do_animate, this);
 
-    // MathJax configuration
+    // MathJax v2 configuration
```

Comment:
Is there a v3 equivalent?

Reply:
Not at the moment it appears: http://docs.mathjax.org/en/latest/output/fonts.html